src/logic/operate.test.js

```diff
@@ -0,0 +1,8 @@
+import operate from './operate';
+
+describe('Addition', () => {
+  test('should Multiply 4 and 7 tobe equal 28', () => {
+    const result = operate(4, 7, 'x');
+    expect(result).toBe('28');
+  });
+});
```

Comment:
- [ ] I would suggest you please add more tests in your operate.test.js as there is only one test that is `Addition`, So I would suggest please test all the functionalities `-, *, ÷, %`